unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/Mentions/Tests/MentionLinkDetectorShould.cs

```diff
@@ -0,0 +1,78 @@
+﻿/*
```

Comment:
we should fix these tests instead of commenting them

Reply:
Correct, I'm waiting to fix them but I would go to the release without them right now and, after that, try to find the problem. These tests are working correctly in Unity editor but, for some reason, are provoking a timeout in the CI.

Reply:
added the TODO comment